src/main/java/net/coderbot/iris/pipeline/newshader/ExtendedShader.java

```diff
@@ -73,6 +73,8 @@ public void apply() {
 
 		if (this.blendModeOverride != null) {
 			this.blendModeOverride.apply();
+		} else if (getName().contains("shadow")) {
+			BlendModeOverride.OFF.apply();
```

Comment:
Would it be possible to handle this where we don't have to check the name of the shader program? If there's a nice way to avoid checking the name, we should do that.

Reply:
Doesn't seem so, unless we want to pass through arguments to say it's a shadow shader.

Reply:
I'd like to find a way to address this before merging.